redhat/patches/0001-dockerfile.patch

```diff
@@ -1,28 +1,43 @@
 diff --git a/Dockerfile b/Dockerfile
-index f2d39ac..4d33566 100644
+index f2d39ac..ab77c76 100644
 --- a/Dockerfile
 +++ b/Dockerfile
 @@ -13,7 +13,7 @@
  # See the License for the specific language governing permissions and
  # limitations under the License.
- 
+
 -FROM golang:1.20.5@sha256:344193a70dc3588452ea39b4a1e465a8d3c91f788ae053f7ee168cebf18e0a50 AS builder
 +FROM registry.access.redhat.com/ubi9/go-toolset@sha256:52ab391730a63945f61d93e8c913db4cc7a96f200de909cd525e2632055d9fa6 AS builder
```

Comment:
Still using the old sha here for the builder image, Also needs to be updated in the other Docker files aswell, other than that looks good :)